Serialization/AbstractFigure.java

```diff
@@ -0,0 +1,20 @@
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Collections;
+
+public abstract class AbstractFigure implements Figure {
+
+    public abstract void draw();
+    public abstract void fill(String color);
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents
first should be fields after constructors and methods